extensions/src/ACRE2TS/inc/public_definitions.h

```diff
@@ -359,6 +359,10 @@ struct FileTransferCallbackExport{
 /*define for file transfer bandwith limits*/
 #define BANDWIDTH_LIMIT_UNLIMITED 0xFFFFFFFFFFFFFFFFll
 
+/*define for metadata tags*/
+#define START_DATA "<ACRE>"
```

Comment:
These defines are only used in `TsCallbacks_static.cpp`. They should be declared as private defines in the cpp file or declared as `static const` in the cpp file.

Reply:
They are not only used in `TsCallback_static.cpp`

Reply:
You are right. Somehow I missed it when reviewing it using the Github interface.